plugins/woocommerce/src/Internal/Features/ProductBlockEditor/ProductTemplates/SimpleProductTemplate.php

```diff
@@ -1104,4 +1105,50 @@ private function add_variation_group_blocks() {
 			)
 		);
 	}
+
+	/**
+	 * Adds the linked products group blocks to the template.
+	 */
+	private function add_linked_products_group_blocks() {
+		if ( ! Features::is_enabled( 'product-virtual-downloadable' ) ) {
```

Comment:
@retrofox @nathanss This is not the right feature flag to validate. Also I didn't see the problem commented there https://github.com/woocommerce/woocommerce/pull/43009/files#r1433178112 already solved here.

Reply:
> @retrofox @nathanss This is not the right feature flag to validate.

🤦 Creating a new PR

> Also I didn't see the problem commented there https://github.com/woocommerce/woocommerce/pull/43009/files#r1433178112 already solved here.

As previously mentioned in the same thread, the issue will be addressed in a follow-up.
After all, isn't the purpose of hiding the implementation behind a feature flag?

Do you see something wrong that could impact the users? I don’t know, something like showing part of this implementation even when the feature flag is desirable?

Reply:
https://github.com/woocommerce/woocommerce/pull/43119